lib/features/help_center/pages/help_center_page.dart

```diff
@@ -0,0 +1,87 @@
+
+import 'package:design_system/design_system.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:grabber/features/help_center/pages/support_page.dart';
+import 'package:grabber/features/shared/bottom_navigation_bar.dart';
+
+import '../../../config/routes/routes.dart';
+import '../../../generated/l10n.dart';
+import '../../help_center/widgets/real_time.dart';
+
+class HelpCenter extends StatelessWidget {
+  const HelpCenter({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      bottomNavigationBar: const GrabberBottomNavigationBar(),
+      body: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          const VerticalGap.xl(),
+          Padding(
+            padding: const EdgeInsets.only(left: 28.0),
+            child: Text(
+              'Central de Ajuda',
+              style: Theme.of(context).textTheme.headlineLarge,
+            ),
+          ),
```

Comment:
This should be done via AppBar attributte on flutter. Please update this

Also do not use hard code values such as 28. Use our DS tokens